hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/jupyter/config/JupyterProcessMgr.java

```diff
@@ -224,14 +227,13 @@ public void stopJupyterServer(Project project, Users user, String hdfsUsername,
     }
     int exitValue = 0;
     Integer id = 1;
-    
     ProcessDescriptor.Builder pdBuilder = new ProcessDescriptor.Builder()
         .addCommand("/usr/bin/sudo")
```

Comment:
do we need sudo here? Same question stands for other parts of this PR where we use sudo

Reply:
As answered to other parts where it was commented on (github is stupidly hiding conversations) we need sudo because we don't want the glassfish user to have the right to start docker instances. This was discussed with @SirOibaf.

Reply:
Thanks, I missed the conversation